git-cliff/src/lib.rs

```diff
@@ -245,8 +242,8 @@ fn process_repository<'a>(
     }
 
     // Print debug information about configuration and arguments.
-    log::trace!("Arguments: {:#?}", args);
-    log::trace!("Config: {:#?}", config);
+    log::debug!("Arguments: {:#?}", args);
+    log::debug!("Config: {:#?}", config);
```

Comment:
I'd like to keep this as trace, otherwise a huge block of text is printed to stderr.

The user also knows the config/args that they provide so I guess it should be fine.